<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InsertionSort</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background:rgb(43,43,43);
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background: rgb(73,73,73);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,1">
    <property name="horizontalSpacing">
     <number>7</number>
    </property>
    <item row="0" column="1" rowspan="2">
     <widget class="QTabWidget" name="arrayTabs">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane{
	border: 2px solid rgb(43,43,43);
	color: white;
	background: rgb(83,83,83)
}
QTabBar::tab{
	background: rgb(83,83,83);
	border: 2px solid rgb(43,43,43);
	border-bottom: none;
	border-top-left-radius: 6px;
	border-top-right-radius: 6px;
	padding: 4px;
	color: white;
	
}
QTabBar::tab:hover{
	background: rgb(103, 103, 103);
}
QTabBar::tab:selected{
	background: rgb(63,63,63);
}
QTabBar::tab:!selected{
	margin-top: 2px;
	margin-right: -1px;
}
 
</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="originalArray">
       <attribute name="title">
        <string>Original array</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" colspan="2">
         <widget class="QTableWidget" name="originalArrayTable">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QHeaderView::section{
	min-width:25px;
	padding: 3px;
	color:white;
	border-right: 2px solid white;
	border-bottom: 2px solid white;
	background: rgb(93,93,93);
}
QHeaderView::section:pressed{
	margin:0;
	border:1px solid rgb(43,43,43);
	
}

QTableWidget{
	gridline-color: white;
	font-size: 12pt;
	background: rgb(73,73,73);
	selection-background-color: rgb(55,55,55);
	color: white;
}

QTableWidget QTableCornerButton::section{
	background: rgb(43,43,43);
}

QTableWidget::item{
	border-bottom: 1px solid rgb(43,43,43);
}

QTableWidget::item:selected{
	color:white;
	selection-background-color: rgb(55,55,55);
}

/*Vertical slider*/
QScrollBar:vertical{
	background: rgb(74, 74, 74);
	border-left: 2px solid rgb(43,43,43);
	border-right: 2px solid rgb(43,43,43);
	border-radius: 0px;
	width: 17px;
	margin: 17px 0 17px 0
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical{
	background:none;
}
QScrollBar::up-arrow:vertical{
	background:none;
	image: url(:/src/Icons/arrowUp.png);	
} 
QScrollBar::down-arrow:vertical{
	background: none;
	image: url(:/src/Icons/arrowDown.png);
}
QScrollBar::handle:vertical{
	background: rgb(115, 115, 115);
	border-radius: 4px;
	min-height: 30px;
	max-height: 30px;
}
QScrollBar::sub-line:vertical{
	background: rgb(74, 74, 74);
	border: 2px solid rgb(43,43,43);
	border-bottom: none;
	height:17px;
	subcontrol-position:top;
	subcontrol-origin:margin;
}
QScrollBar::sub-line:vertical:pressed{
	background: rgb(54, 54, 54);
}
QScrollBar::add-line:vertical{
	background: rgb(74, 74, 74);
	border: 2px solid rgb(43,43,43);
	border-top: none;
	height:17px;
	subcontrol-position:bottom;
	subcontrol-origin:margin;
}
QScrollBar::add-line:vertical:pressed{
	background: rgb(54, 54, 54);
}

/*Horizontal slider*/

QScrollBar:horizontal{
	background: rgb(74, 74, 74);
	border-top: 2px solid rgb(43,43,43);
	border-bottom: 2px solid rgb(43,43,43);
	border-radius: 0px;
	height: 17px;
	margin: 0 17px 0 17px;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal{
	background:none;
}
QScrollBar::left-arrow:horizontal{
	background:none;
	image: url(:/src/Icons/arrowLeft.png);	
} 
QScrollBar::right-arrow:horizontal{
	background: none;
	image: url(:/src/Icons/arrowRight.png);	
}
QScrollBar::handle:horizontal{
	background: rgb(115, 115, 115);
	border-radius: 4px;
	min-width: 30px;
	max-width: 600px;
}
QScrollBar::sub-line:horizontal{
	background: rgb(74, 74, 74);
	border: 2px solid rgb(43,43,43);
	border-right: none;
	width:17px;
	subcontrol-position:left;
	subcontrol-origin:margin;
}
QScrollBar::sub-line:horizontal:pressed{
	background: rgb(54, 54, 54);
}
QScrollBar::add-line:horizontal{
	background: rgb(74, 74, 74);
	border: 2px solid rgb(43,43,43);
	border-left: none;
	width:17px;
	subcontrol-position:right;
	subcontrol-origin:margin;
}
QScrollBar::add-line:horizontal:pressed{
	background: rgb(54, 54, 54);
}</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="autoScroll">
           <bool>false</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <property name="rowCount">
           <number>9</number>
          </property>
          <property name="columnCount">
           <number>10</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>84</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>36</number>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="sortArrayBtn">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background: rgb(83, 83, 83);
	border: 2px solid rgb(43,43,43);
	border-radius: 6px;
	color: white;
	margin: 5%;
	padding: 5px;
	spacing: 3px;
	font: 700 10pt &quot;Arial&quot;;
}

QPushButton:hover{
	background: rgb(103, 103, 103);
}

QPushButton:pressed{
	background: rgb(53,53,53);
}

</string>
          </property>
          <property name="text">
           <string>Sort array</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sortedArray">
       <attribute name="title">
        <string>Sorted array</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="sortedArrayTable">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QHeaderView::section{
	min-width:25px;
	padding: 3px;
	color:white;
	border-right: 2px solid white;
	border-bottom: 2px solid white;
	background: rgb(93,93,93);
}
QHeaderView::section:pressed{
	margin:0;
	border:1px solid rgb(43,43,43);
	
}

QTableWidget{
	gridline-color: white;
	font-size: 12pt;
	background: rgb(73,73,73);
	selection-background-color: rgb(55,55,55);
	color: white;
}

QTableWidget QTableCornerButton::section{
	background: rgb(43,43,43);
}

QTableWidget::item{
	border-bottom: 1px solid rgb(43,43,43);
}

QTableWidget::item:selected{
	color:white;
	selection-background-color: rgb(55,55,55);
}

/*Vertical slider*/
QScrollBar:vertical{
	background: rgb(74, 74, 74);
	border-left: 2px solid rgb(43,43,43);
	border-right: 2px solid rgb(43,43,43);
	border-radius: 0px;
	width: 17px;
	margin: 17px 0 17px 0;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical{
	background:none;
}
QScrollBar::up-arrow:vertical{
	background:none;
	image: url(:/src/Icons/arrowUp.png);	
} 
QScrollBar::down-arrow:vertical{
	background: none;
	image: url(:/src/Icons/arrowDown.png);
}
QScrollBar::handle:vertical{
	background: rgb(115, 115, 115);
	border-radius: 4px;
	min-height: 30px;
	max-height: 30px;
}
QScrollBar::sub-line:vertical{
	background: rgb(74, 74, 74);
	border: 2px solid rgb(43,43,43);
	border-bottom: none;
	height:17px;
	subcontrol-position:top;
	subcontrol-origin:margin;
}
QScrollBar::sub-line:vertical:pressed{
	background: rgb(54, 54, 54);
}
QScrollBar::add-line:vertical{
	background: rgb(74, 74, 74);
	border: 2px solid rgb(43,43,43);
	border-top: none;
	height:17px;
	subcontrol-position:bottom;
	subcontrol-origin:margin;
}
QScrollBar::add-line:vertical:pressed{
	background: rgb(54, 54, 54);
}

/*Horizontal slider*/

QScrollBar:horizontal{
	background: rgb(74, 74, 74);
	border-top: 2px solid rgb(43,43,43);
	border-bottom: 2px solid rgb(43,43,43);
	border-radius: 0px;
	height: 17px;
	margin: 0 17px 0 17px;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal{
	background:none;
}
QScrollBar::left-arrow:horizontal{
	background:none;
	image: url(:/src/Icons/arrowLeft.png);	
} 
QScrollBar::right-arrow:horizontal{
	background: none;
	image: url(:/src/Icons/arrowRight.png);	
}
QScrollBar::handle:horizontal{
	background: rgb(115, 115, 115);
	border-radius: 4px;
	min-width: 30px;
	max-width: 600px;
}
QScrollBar::sub-line:horizontal{
	background: rgb(74, 74, 74);
	border: 2px solid rgb(43,43,43);
	border-right: none;
	width:17px;
	subcontrol-position:left;
	subcontrol-origin:margin;
}
QScrollBar::sub-line:horizontal:pressed{
	background: rgb(54, 54, 54);
}
QScrollBar::add-line:horizontal{
	background: rgb(74, 74, 74);
	border: 2px solid rgb(43,43,43);
	border-left: none;
	width:17px;
	subcontrol-position:right;
	subcontrol-origin:margin;
}
QScrollBar::add-line:horizontal:pressed{
	background: rgb(54, 54, 54);
}</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhPreferNumbers</set>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="autoScroll">
           <bool>false</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <property name="rowCount">
           <number>10</number>
          </property>
          <property name="columnCount">
           <number>10</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>84</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>36</number>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="arrayList">
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
	border: 3px solid rgb(43,43,43);
	background:rgb(45,45,45);
	color:white;
	/*font: 11pt &quot;Arial&quot;;*/
}
QListWidget::item{
	height: 35px;
	padding: 5px;
	margin: 0 0 5px 0;
	border-radius: 3;
	background:rgb(73,73,73);
}
QListWidget::item:selected{
	background: rgb(55,55,55);
	border: 2px solid rgb(85, 85, 85);
	border-radius: 3;
	color:white;
}
QListWidget::item:hover:!selected{
	background: rgb(85, 85, 85);
}
/*Vertical slider*/
QScrollBar:vertical{
	background: rgb(74, 74, 74);
	border: none;
	border-left: 4px solid rgb(43,43,43);
	border-radius: 0px;
	width: 17px;
	margin: 17px 0 17px 0
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical{
	background:none;
}
QScrollBar::up-arrow:vertical{
	background:none;
	image: url(:/src/Icons/arrowUp.png);	
} 
QScrollBar::down-arrow:vertical{
	background: none;
	image: url(:/src/Icons/arrowDown.png);
}
QScrollBar::handle:vertical{
	background: rgb(115, 115, 115);
	border-radius: 4px;
	min-height: 30px;
	/*Add max value*/
}
QScrollBar::sub-line:vertical{
	background: rgb(74, 74, 74);
	border: none;
	border-left: 4px solid rgb(43,43,43);
	height:17px;
	subcontrol-position:top;
	subcontrol-origin:margin;
}
QScrollBar::sub-line:vertical:pressed{
	background: rgb(54, 54, 54);
}
QScrollBar::add-line:vertical{
	background: rgb(74, 74, 74);
	border: none;
	border-left: 4px solid rgb(43,43,43);
	height:17px;
	subcontrol-position:bottom;
	subcontrol-origin:margin;
}
QScrollBar::add-line:vertical:pressed{
	background: rgb(54, 54, 54);
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="selectionRectVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="Add_DeleteLayout">
      <item>
       <widget class="QLabel" name="arrayListLabel">
        <property name="styleSheet">
         <string notr="true">QLabel{
	color: white;
	padding: 5px;
	font: 700 14pt &quot;Arial&quot;;
}</string>
        </property>
        <property name="text">
         <string>List of arrays:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addArrayBtn">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background: rgb(83, 83, 83);
	border: 2px solid rgb(43,43,43);
	border-radius: 6px;
	color: white;
	padding: 5px;
	min-width: 50px;
	margin: 5%;
	font: 700 9pt &quot;Arial&quot;;
}

QPushButton:hover{
	background: rgb(103, 103, 103);
}

QPushButton:pressed{
	background: rgb(53,53,53);
}

</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteArrayBtn">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background: rgb(83, 83, 83);
	border: 2px solid rgb(43,43,43);
	border-radius: 6px;
	color: white;
	padding: 5px;
	min-width: 50px;
	margin: 5%;
	font: 700 9pt &quot;Arial&quot;;
}

QPushButton:hover{
	background: rgb(103, 103, 103);
}

QPushButton:pressed{
	background: rgb(53,53,53);
}

</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1187</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar{
	background: rgb(83, 83, 83); 
	spacing: 5px; 
	color: white;
	border-bottom:2px solid rgb(43,43,43);
	padding: 3px;
	border-radius: 4px;
}
QMenuBar::item:selected {
	background: rgb(75, 75, 75); 
	border: 1px solid rgb(150, 150, 150); 
	border-radius: 4px;
}
QMenuBar::item:pressed {
	background: rgb(68, 68, 68);
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true">QMenu{
	background: rgb(80, 80, 80); 
	color: white; 
}
QMenu::item:selected{
	background: rgb(75, 75, 75);
}
QMenu::item:pressed{
	background: rgb(68, 68, 68);
}</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveChanges"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTests">
    <property name="styleSheet">
     <string notr="true">QMenu{
	background: rgb(80, 80, 80); 
	color: white; 
}
QMenu::item:selected{
	background: rgb(75, 75, 75);
}
QMenu::item:pressed{
	background: rgb(68, 68, 68);
}</string>
    </property>
    <property name="title">
     <string>Tests</string>
    </property>
    <addaction name="actionLaunch_tests"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="styleSheet">
     <string notr="true">QMenu{
	background: rgb(80, 80, 80); 
	color: white; 
}
QMenu::item:selected{
	background: rgb(75, 75, 75);
}
QMenu::item:pressed{
	background: rgb(68, 68, 68);
}</string>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_application"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTests"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar{
	background: rgb(73,73,73);
	border-top: 2px solid rgb(43,43,43);
}
QStatusBar::item{
	border: none;
}
QStatusBar QProgressBar{
	border: 2px solid rgb(43,43,43); 
	background: rgb(143,143,143);
	border-radius: 4px; 
	color: white; 
	text-align: center;
}
QStatusBar QProgressBar::chunk{
	background: #ac68cc
}
</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionSaveChanges">
   <property name="text">
    <string>Save changes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLaunch_tests">
   <property name="text">
    <string>Launch tests</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAbout_application">
   <property name="text">
    <string>About application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
